### PR TITLE
fixed load_image_into_numpy_array to handle png images

### PR DESCRIPTION
**Pull request by pinkbunny1**
load_image_into_numpy_array(image) raises ValueError when the given image format is png.
The workaround I found is to convert the image to 'RGB'.

### CODE REVIEW

**Comment by pinkbunny1** [comment]
I signed it!

**Comment by pinkbunny1** [comment]
Hi @pkulzc  I have removed the unnecessary metadata changes.

File: research/object_detection/object_detection_tutorial.ipynb [hunk 1]
```diff
@@ -335,7 +337,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython2",
-   "version": "2.7.10"
+   "version": "2.7.14"
```
Comment (pkulzc) [inline research/object_detection/object_detection_tutorial.ipynb]: Please also revert this version.

**Comment by pinkbunny1** [comment]
@pkulzc BTW do you know why CLA check is failed despite I have signed the agreement ?

**Comment by pkulzc** [comment]
Looks like you are using different accounts for commit and pull request.

**Comment by pinkbunny1** [comment]
@pkulzc Opps. That account was an empty account which I was not aware I had, so I deleted it after I saw it had contributed to this commit. What should I do... ?

**Comment by pkulzc** [comment]
You can revert your commits, check your git config and re-commit the changes. Use stackoverflow or google search if you don't know the commands to do those.

**Comment by pinkbunny1** [comment]
@pkulzc Thanks for the help. I followed your instructions but still CLA check has failed. Any ideas as to why ?

**Comment by pkulzc** [comment]
You should "git reset" your branch instead of committing an extra "revert commit". Besides, the first 4 commits are still from different accounts. Please check git documentations for command to use.

**Pull request closed without merging**